service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseControllerTest.java

```diff
@@ -645,4 +645,105 @@ private HearingBooking hearing(UUID cmoId) {
                 .build();
         }
     }
+
+    @Nested
+    class Fpla2885 {
+        String migrationId = "FPLA-2885";
+        UUID idOne = UUID.randomUUID();
+        UUID idTwo = UUID.randomUUID();
+        UUID idThree = UUID.randomUUID();
+        UUID idFour = UUID.randomUUID();
+
+        @Test
+        void shouldMigrateExpectedListElementCodes() {
+            Element<HearingBooking> hearingOne
+                = element(idOne, hearingBookingWithCancellationReason("OT8"));
+
+            Element<HearingBooking> hearingTwo
+                = element(idTwo, hearingBookingWithCancellationReason("OT9"));
+
+            Element<HearingBooking> hearingThree
+                = element(idThree, hearingBookingWithCancellationReason("OT10"));
+
+            Element<HearingBooking> hearingFour
+                = element(idFour, hearingBookingWithCancellationReason("OT7"));
+
+            List<Element<HearingBooking>> cancelledHearingBookings = List.of(
+                hearingOne, hearingTwo, hearingThree, hearingFour);
+
+            CaseDetails caseDetails = caseDetails(migrationId, cancelledHearingBookings);
+            CaseData extractedCaseData = extractCaseData(postAboutToSubmitEvent(caseDetails));
+
+            assertThat(extractedCaseData.getCancelledHearingDetails()).isEqualTo(List.of(
```

Comment:
better to use:
 assertThat(extractedCaseData.getCancelledHearingDetails()).containsExactly(e1,e2)
 it gives better report when assertion fails